src/components/FormPage.tsx

```diff
@@ -36,28 +36,12 @@ function FormPage({ grunnbelop }: FormPageProps) {
 
   const lastPage = <BeregnPage key="beregn" />
 
-  const { curStep, step, goToNext, goBack, goTo } = useMultiStepForm(
+  const { curStep, step, goToNext, goBack } = useMultiStepForm(
```

Comment:
Det er muligens her det er uklart for meg. Hvis `useMultiStepForm` er den som holder tilstanden over navigeringen. Hva brukes `FormPageProps` til, og er den nødvendig?